content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -471,36 +470,35 @@ <h2 id="toc-webaudio-api">Using getUserMedia with the Web Audio API</h2>
 </code></pre>
 <p>Demos:</p>
 <ul>
-<li><a href="http://webaudiodemos.appspot.com/input/index.html">Live Input Visualizer</a></li>
-<li><a href="http://webaudiodemos.appspot.com/AudioRecorder/index.html">Audio Recorder</a></li>
-<li><a href="http://webaudiodemos.appspot.com/pitchdetect/index.html">Pitch Detector</a></li>
+<li><a href="https://webaudiodemos.appspot.com/input/index.html">Live-input visualizer</a></li>
+<li><a href="https://webaudiodemos.appspot.com/AudioRecorder/index.html">Audio recorder</a></li>
+<li><a href="https://webaudiodemos.appspot.com/pitchdetect/index.html">Pitch detector</a></li>
 </ul>
-<p>For more information, see <a href="http://updates.html5rocks.com/2012/09/Live-Web-Audio-Input-Enabled">Chris Wilson's post</a>.</p>
+<p>For more information, see <a href="https://updates.html5rocks.com/2012/09/Live-Web-Audio-Input-Enabled">Chris Wilson's post</a>.</p>
 <h2 id="toc-conclusion">Conclusion</h2>
 
-<p>Historically, device access on the web has been a tough nut to crack. <a href="http://www.slideshare.net/jamesgpearce/mobile-device-apis">Many tried, few succeeded</a>. Most of the early ideas never gained widespread adoption or took hold outside of a proprietary environment. Perhaps the main problem has been that the web's security model is <em>very</em> different from the native world. In particular, you probably don't want every Joe Shmoe website to have random access to your video camera or microphone. It's been difficult to get right.</p>
+<p>Historically, device access on the web has been a tough nut to crack. <a href="https://www.slideshare.net/jamesgpearce/mobile-device-apis">Many tried, few succeeded</a>. Most of the early ideas never gained widespread adoption or took hold outside of a proprietary environment. Perhaps the main problem has been that the web's security model is <em>very</em> different from the native world. In particular, you probably don't want every Joe Shmoe website to have random access to your video camera or microphone. It's been difficult to get right.</p>
```

Comment:
Please wrap under 80 characters per line

Reply:
Nope?

Reply:
Do you know how I can do this? I see an option to "soft wrap," but does that automatically wrap under 80 characters per line?

Reply:
Done.